Sources/Java2SwiftLib/JavaClassTranslator.swift

```diff
@@ -582,10 +612,33 @@ extension JavaClassTranslator {
     )
     let fieldAttribute: AttributeSyntax = javaField.isStatic ? "@JavaStaticField" : "@JavaField";
     let swiftFieldName = javaField.getName().escapedSwiftName
-    return """
+
+    if let optionalType = swiftTypeName.optionalWrappedType() {
+      let setter = if !javaField.isFinal {
+      """
+      
+        set {
+          \(swiftFieldName) = newValue?.toJavaOptional()
+        }
+      """
+      } else { "" }
+      return """
+      \(fieldAttribute)(isFinal: \(raw: javaField.isFinal))
+      public var \(raw: swiftFieldName): \(raw: typeName)
+
+      
+      public var \(raw: swiftFieldName)Optional: \(raw: optionalType) {
+        get {
+          Optional(javaOptional: \(raw: swiftFieldName))
+        }\(raw: setter)
+      }
+      """
```

Comment:
I think going the opposite way maybe is more typical? It is "normal" to expose the  Swift optional and maybe have the java optional type property as `thingJava` or some `thing$raw` or `thing$java`? Just ideas 🤔

Reply:
I went the opposite way first but then ran into a roadblock since the JavaMethod macro doesn't take in a name to use as the Java method name (which can be done, just felt weird as part of this pr). This means that it _needs_ to have the exact Java name and then it felt weird to not have parity between fields and methods with optional mapping

Reply:
We can go with this way if you get it to build tbh, thanks for the background info